pypeit/spectrographs/ntt_efosc2.py

```diff
@@ -0,0 +1,336 @@
+
+        #edge parameters
+        par['calibrations']['slitedges']['edge_thresh'] = 75.
+
+        # Tilt parameters
+        par['calibrations']['tilts']['tracethresh'] = 25.0
+        par['calibrations']['tilts']['spat_order'] = 3
+        par['calibrations']['tilts']['spec_order'] = 4
+        
+        # Image processing
+        # The overscan region might cause oversubtraction of the background, set it to False
+        par['scienceframe']['process']['use_overscan'] = False 
+
+        # 1D wavelength solution
+        par['calibrations']['wavelengths']['method'] = 'full_template'
+        par['calibrations']['wavelengths']['lamps'] = ['HeI', 'ArI']
+        par['calibrations']['wavelengths']['rms_threshold'] = 0.25
+        par['calibrations']['wavelengths']['sigdetect'] = 10.0
+        par['calibrations']['wavelengths']['fwhm'] = 4.0
+        par['calibrations']['wavelengths']['n_final'] = 4
+
+        # Flats
+        par['calibrations']['flatfield']['tweak_slits_thresh'] = 0.90
+        par['calibrations']['flatfield']['tweak_slits_maxfrac'] = 0.10
+        
+        # Extraction
+        par['reduce']['skysub']['bspline_spacing'] = 0.8
+        par['reduce']['skysub']['no_poly'] = True
+        par['reduce']['skysub']['bspline_spacing'] = 0.6
+        par['reduce']['skysub']['joint_fit'] = False
+        par['reduce']['skysub']['global_sky_std']  = False
+
+        par['reduce']['extraction']['sn_gauss'] = 4.0
+        par['reduce']['findobj']['sig_thresh'] = 5.0
+        par['reduce']['skysub']['sky_sigrej'] = 5.0
+        par['reduce']['findobj']['find_trim_edge'] = [5,5]
+
+        return par
+
+    def config_specific_par(self, scifile, inp_par=None):
+        """
+        Modify the ``PypeIt`` parameters to hard-wired values used for
+        specific instrument configurations.
+
+        Args:
+            scifile (:obj:`str`):
+                File to use when determining the configuration and how
+                to adjust the input parameters.
+            inp_par (:class:`~pypeit.par.parset.ParSet`, optional):
+                Parameter set used for the full run of PypeIt.  If None,
+                use :func:`default_pypeit_par`.
+
+        Returns:
+            :class:`~pypeit.par.parset.ParSet`: The PypeIt parameter set
+            adjusted for configuration specific parameter values.
+        """
+        # Start with instrument wide
+        par = super().config_specific_par(scifile, inp_par=inp_par)
+
+        # Wavelength calibrations
+        if self.get_meta_value(scifile, 'dispname') == 'Gr#6':
+            par['calibrations']['wavelengths']['reid_arxiv'] = 'ntt_efosc2_Gr6.fits'
+        elif self.get_meta_value(scifile, 'dispname') == 'Gr#5':
+            par['calibrations']['wavelengths']['reid_arxiv'] = 'ntt_efosc2_Gr5.fits'
+            # Fringes are affecting this Grism significantly, skip flat fielding
```

Comment:
Ok.  Please generate a brief `ntt_efosc2.rst` file in `docs/` and add some notes about this (and whatever else you wish).  You can use `keck_lris.rst` as an example.

Reply:
Done.